src/frontend/js/api/lms/openedx-fonzie.spec.ts

```diff
@@ -30,6 +31,17 @@ describe('Fonzie API', () => {
     await expect(api.user.me()).resolves.toEqual(user);
   });
 
+  it('uses its own route to get user profile', async () => {
+    const openEdxProfile = OpenEdxApiProfileFactory().one();
+    fetchMock.get(
+      `https://demo.endpoint.api/api/user/v1/accounts/${openEdxProfile.username}`,
```

Comment:
@jbpenrath 
Why does this route use `api/user/v1/` where other api route use `/api/v1.0/user` ?

Reply:
`/api/v1.0/user` are routes developed by us in fonzie so we follow versionned route pattern we are using in our projects.

`api/user/v1/` are routes of OpenEdX.